Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionDataRecord.mm

```diff
@@ -234,7 +241,17 @@
         EXPECT_EQ(dataRecords[0].totalSize + dataRecords[1].totalSize, 237UL);
 
         [testController removeDataOfTypes:allDataTypesSet forDataRecords:dataRecords completionHandler:^{
+            EXPECT_NS_EQUAL(dataRecords[0].errors, @[ ]);
+            EXPECT_NS_EQUAL(dataRecords[1].errors, @[ ]);
+
+            if (dataRecords[0].errors.count)
+                EXPECT_NS_EQUAL(dataRecords[0].errors[0].userInfo[NSDebugDescriptionErrorKey], @"");
```

Comment:
trying to see if there is a `NSDebugDescriptionError` that can be used to figure out why this is failing on the bots. will remove before landing.

Reply:
didn't see any errors, but i think the issue was that i forgot to also make `RemoveDataRecordsForMultipleContexts` use `dataRecordTestTwoManifest`, only `GetDataRecordsForMultipleContexts` was using it.

Reply:
wasn't able to figure out why `RemoveDataRecordsForMultipleContexts` is failing on the bots, but not locally. Will keep this disabled and will only enable `RemoveDataRecords`.